app/src/main/java/com/moodup/movies/viewmodel/MovieViewModel.kt

```diff
@@ -4,20 +4,23 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.moodup.movies.model.Movie
 import com.moodup.movies.model.Result
-import com.moodup.movies.repository.api.MovieRepository
+import com.moodup.movies.repository.api.call.MovieRepository
 import com.moodup.movies.state.UIState
 import retrofit2.Call
 import retrofit2.Callback
+import retrofit2.http.Query
 
 class MovieViewModel : ViewModel() {
     private var movieRepository: MovieRepository = MovieRepository()
 
+    var totalItemCount : Int = 0
+    var isDataLoading:Boolean = false
     var movieLiveData = MutableLiveData<List<Movie>>()
     var UIstateLiveData = MutableLiveData<UIState>(UIState.INITIALIZED)
 
-    fun getMovies(query : String) {
-        UIstateLiveData.postValue(UIState.LOADING)
-            movieRepository.getAllMovies(query).enqueue(object : Callback<Result> {
+    fun getMovies(totalPageNumber: Int , query : String) {
```

Comment:
Możesz zmienić tę nazwę na currentPageNumber czy coś takiego bo chyba o to tutaj chodzi :)